extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php

```diff
@@ -84,8 +85,7 @@ public function index() {
 		// theme designer application theme settings
 		$this->applicationThemeSettings = SassUtil::getApplicationThemeSettings();
 
-		// use either wikia.com (production) or wikia-staging.com (staging)
-		$this->wikiaBaseDomain = $wgWikiaBaseDomain;
+		$this->wikiBaseDomain = $wgCreateEnglishWikisOnFandomCom ? $wgFandomBaseDomain : $wgWikiaBaseDomain;
```

Comment:
It was `this->wikiaBaseDomain` before. Is that change correct? Is it even used anywhere?

Reply:
Ah, I see it's here https://github.com/Wikia/app/pull/15930/files#diff-cdb6d8de3a494b9a55793be8c4dd81efR34
Magic methods :disappointed: